src/index.js

```diff
@@ -1,42 +1,34 @@
-import _ from "lodash";
-import "./style.css";
+import _ from 'lodash';
+import './style.css';
 
-const taskArray = [
-  {
-    index: 0,
-    descriptions: "I  will code",
-    completed: true,
-  },
-  {
-    index: 1,
-    descriptions: "i love code",
-    completed: false,
-  },
-  {
-    index: 2,
-    descriptions: "i like webpack",
-    completed: true,
-  },
-  {
-    index: 4,
-    description: "I love microverse",
-    completed: false,
-  },
-];
+import addedTask from "./module/class.js"
+import { addToLocalStorage, getFromLocalStorage } from "./module/localstorage.js"
+import displayList from "./module/displaylist.js"
```

Comment:
- [ ] in these lines you are importing from the module folder however the folder doesn't exist it looks like you might have forgotten to push these changes, therefore the user receives below error.

![afbeelding](https://user-images.githubusercontent.com/39240820/194628081-52c65874-3b5d-4822-b2a8-7cb76e69366e.png)